<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hilfe</class>
 <widget class="QMainWindow" name="hilfe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QtLog Hilfe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="ButtonEnde">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>175</red>
            <green>215</green>
            <blue>217</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>175</red>
            <green>215</green>
            <blue>217</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>175</red>
            <green>215</green>
            <blue>217</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>( F10 )  Ende</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>   Hilfe Katalog</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>244</red>
                <green>244</green>
                <blue>211</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>244</red>
                <green>244</green>
                <blue>211</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <item>
           <property name="text">
            <string> ----- Index  ---------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Configuriere</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Betriebsarten</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qso Verwaltung</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qso Eingabe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qso Contest</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qsl Eingang</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qsl Druck</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Qsl Text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DataExport</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CwDaemon</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DxSpot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rig Control-Hamlib</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DX-LaenderListe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DX-SubPrefixListe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DOK Verwaltung</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>QRZ Callbook</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Awd-Status</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DB-Administration</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Installation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>------------------------</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Historie</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>   Funktions Beschreibung in Kurzform</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="font">
           <font>
            <family>Thorndale AMT</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
